src/Extractor/RpcExtractor.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimPod\JsonRpc\Extractor;
+
+use Psr\Http\Message\MessageInterface;
+use function Safe\json_decode;
+
+abstract class RpcExtractor
+{
+    /**
+     * @return mixed
+     */
+    protected static function getFieldFromBody(MessageInterface $message, string $key)
+    {
+        $body = $message->getBody();
+        $rpc  = json_decode($body->getContents(), true);
```

Comment:
špatný design ... budeš dělat json_decode pro každý field? to nedává smysl ...

a nějak nechápu proč by tohle mělo být statické...

když už chceš mít tuhle logiku nějak separe, tak bych to udělal tak, že rpcExtractor přijme messageInterface v konstruktoru, tam udělá decode a zbytek metod bude pracovat už nad tím pole